nemo/collections/asr/modules/rnnt.py

```diff
@@ -1579,17 +1580,15 @@ def needs_labels(self):
 
     @property
     def input_types(self):
-        return OrderedDict(
-            {
-                "encoder_output": NeuralType(('B', 'D', 'T'), AcousticEncodedRepresentation()),
-                "targets": NeuralType(('B', 'T'), LabelsType()),
-                "target_lengths": NeuralType(tuple('B'), LengthsType()),
-            }
-        )
+        return {
```

Comment:
Why to drop the OrderedDict?

Reply:
Python dicts are orderd dics by default since 3.7+